ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModuleConstants.java

```diff
@@ -81,7 +81,7 @@
         .build();
   }
 
-  public static Map<String, Object> getConstants() {
+  public static Map<String, Object> getConstants(float fontScale) {
```

Comment:
In [this earlier comment](https://github.com/facebook/react-native/pull/11008#issuecomment-263545377) you mentioned that we should get the font scale from `getResources().getConfiguration().fontScale` rather than calculating it from `scaledDensity / density`.

The reason I pass `fontScale` around rather than the Configuration object is that I wanted it to be very obvious that we are passing `mFontScale` to JavaScript. I was concerned that there might be some race where font scale is extracted from the Configuration object but it has a different value than `mFontScale`.